packages/dds/tree/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -1548,7 +1549,7 @@ describe("ModularChangeFamily", () => {
 		}
 
 		const sessionId = "session1" as SessionId;
-		const context: ChangeEncodingContext = { originatorId: sessionId };
+		const context: ChangeEncodingContext = { originatorId: sessionId, revision: undefined };
```

Comment:
Would be good to add a test that takes a context with a defined revision.

Reply:
Actually, scratch that, it's probably best to just add it to the revisionTagCodec.spec.ts.

Reply:
Updated